src/components/typography/text/text.js

```diff
@@ -49,6 +49,7 @@ Headline.propTypes = {
   title: nonEmptyString,
   truncate: PropTypes.bool,
   intlMessage: requiredIf(intlMessageShape, props => !props.children),
```

Comment:
BTW: This has caused breaking tests in the MC. Maybe it's an overly eager prop-type we could also remove entirely. As before you could have empty children. Now you can't, you have to have a `intlMessage` if no `children` are set which is a bit hard as a validation.

Reply:
When would you have empty children? Does that even make sense? 🤔

Reply:
I am not saying it does make sense. I am saying we do have that and this was a breaking change as a result cause we allowed it before and then didn't.

Reply:
I saw the PR now. I think it does not make sense to render the Text component with empty children. We should just not render the component in the first place.

```js
const title = undefined;

// wrong
<Text.Body>{title}</Text.Body>

// correct
{title && (<Text.Body>{title}</Text.Body>)}
```

WDYT?

Reply:
Yes it would make sense. However people didn't do that 🗡